docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md

```diff
@@ -78,7 +78,7 @@ The following instructions use user-level systemd to provide the necessary acces
 
 ## Verify the python version installed on the node
 
-Verify that Python 3.8-3.9 is installed on the node. v3.8 is recommended.
+Verify that Python 3.6-3.12 is installed on the node. v3.6 is recommended.
```

Comment:
I would leave 3.8 as recommended, and honestly just keep this as 3.8+ I think that's going to be more correct going forward. If somebody complains we can tell them 3.6 still should work but I don't think we want to officially document supporting it bc then people will try to install it.